### DISCUSSION TITLE
How to use `past_feat_dynamic_real` ?

### DISCUSSION THREAD

**Discussion by sayedathar11** [awslabs/gluonts#1720] [Q&A]
We are using GluonTS Deepar model to predict  target at a specific timeframe , we have a feature which is numerical . We compute some numerical stats on this feature and we can use this to predict Future Values of Target .

But we now realized that values of this feature won't be available in Future when we have to predict the target . We can't use `feat_dynamic_real` and as per our understanding we have to use `past_feat_dynamic_real` and instead use past stats values of this numerical feature to predict future value .

Whether this understanding is correct , also can you please add documentation regarding how to use `past_feat_dynamic_real` .

**Comment by pratikgehlott** [comment]
Hi @lostella , @mbohlkeschneider I am also facing the same doubt how are we suppose to use past_feat_dynamic_real ?

**Comment by mbohlkeschneider** [comment]
Hi everyone, 

thank you for your questions. `past_feat_dynamic_real` is still not officially supported for DeepAR. There is a [PR](https://github.com/awslabs/gluon-ts/pull/1757) but it is not yet merged.

**Comment by spunknic** [comment]
I have the same doubt. I mean, in the real world we will never have future observations of any feature

**Reply by jimbow973** [reply]
why never? let's say  you forecast sales and you know already the discount periods/discount amount etc. as future predictor. I think we can find many applications.
